README.md

```diff
@@ -1,381 +1 @@
-#   you may not use this file except in compliance with the License.
-#   You may obtain a copy of the License at
-#
-#   http://www.apache.org/licenses/LICENSE-2.0
-#
-#   Unless required by applicable law or agreed to in writing, software
-#   distributed under the License is distributed on an \"AS IS\" BASIS,
-#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#   See the License for the specific language governing permissions and
-#   limitations under the License.
-################################################################################
- # Carbon Configuration Parameters
-wso2.carbon:
-   # value to uniquely identify a server
- id: carbon-kernel
-   # server name
- name: WSO2 Carbon Kernel
- tenant: default
-   # ports used by this server
- ports:
-     # port offset
-   offset: 0
-   # StartupOrderResolver related configurations
- startupResolver:
-   capabilityListenerTimer:
-       # delay in milliseconds before task is to be executed
-     delay: 20
-       # time in milliseconds between successive task executions
-     period: 20
-   pendingCapabilityTimer:
-       # delay in milliseconds before task is to be executed
-     delay: 60000
-       # time in milliseconds between successive task executions
-     period: 30000
-   # JMX Configuration
- jmx:
-     # To enable JMX Monitoring, change this value to true
-   enabled: false
-     # Server HostName
-   hostName: 127.0.0.1
-     # The port RMI server should be exposed
-   rmiServerPort: 11111
-     # The port RMI registry is exposed
-   rmiRegistryPort: 9999
-   ```
-
-## Step 4: Updating the default configurations
-
-If you need to override the default values, you need to copy  the configuration segment to the `deployment.yaml` file and change the value. The server will pick the new configs from the `deployment.yaml` after restarting the server. For example, if you need to change the update interval of the carbon deployment engine to 20, you need to copy the configuration segment from the configuration document of the component to the `deployment.yaml` file of the server.
-
-Copy the `updateInterval` configuration to the `deployment.yaml` file and change the value to 20, as shown below.
-
-```yaml
-# Deployment Engine related configurations
-wso2.deployment:
-   # Deployment update interval in seconds. This is the interval between repository listener
-   # executions.
- updateInterval: 20
- ```
-
-The updated `deployment.yaml` file will be as follows:
-
-```yaml
-# Carbon Configuration Parameters
-wso2.carbon:
-   # value to uniquely identify a server
- id: carbon-kernel
-   # server name
- name: WSO2 Carbon Kernel
-   # ports used by this server
- ports:
-     # port offset
-   offset: 0
-
-# Deployment Engine related configurations
-wso2.deployment:
-   # Deployment update interval in seconds. This is the interval between repository listener
-   # executions.
- updateInterval: 20
- ```
\ No newline at end of file
+#Carbon Configuration
```

Comment:
@imesh, send a new PR with README and docs changes #18 . Can you please review and merge it